src/EFCore.PG.NTS/Extensions/NpgsqlNetTopologySuiteGeometryExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+
+// ReSharper disable once CheckNamespace
+namespace NetTopologySuite.Geometries
+{
+    /// <summary>
+    /// Provides additional geometry methods which are not in NetTopologySuite.
+    /// </summary>
+    public static class NpgsqlNetTopologySuiteGeometryExtensions
```

Comment:
okay, I will move it. But I am looking more on NpgsqlNetTopologySuiteMethodCallTranslatorPlugin. I am not sure if implement translation of my new extension functions either in same way as Transform (simply calling _sqlExpressionFactory.Function) or using TranslateGeometryFunction (as it is in my first commit). Because I do not fully understand the logic behind inferring, finding and applying mappings there. Can you please give me some hint?

Reply:
TranslateGeometryMethod is intended for translating methods on the NetTopologySuite type itself (e.g. Geometry), whereas TranslateDbFunction is for translating extension methods we add in NpgsqlNetTopologySuiteDbFunctionsExtensions; so your translation should go in the latter. But the difference isn't very important - in the end they both end up calling _sqlExpressionFactory.Function.